src/components/packages/Packages-outline.svelte

```diff
@@ -0,0 +1,93 @@
+<script>
+    import {fade} from 'svelte/transition';
+    export let photos, name, length, price, outfits, location;
+
+    let package_item = 0;
+
+    let visible = true;
+
+    function next(){
+        if(package_item === (packages.length-1)){
+            package_item = 0;
+        }else{
+            package_item++
+        }
+         
+        visible = false;
+    }
+
+    function previous(){
+        if(package_item === 0){
+            package_item = (packages.length-1);
+        }else{
+            package_item--;
+        }
+         
+        visible = false;
+    }
+
+    
+</script>
+<style>
+    h1{
+        /* font-family: 'Permanent Marker', cursive; */
+        font-family: 'Vollkorn SC', serif;
```

Comment:
I had font utility classes that you can use. Check out `preset-description.svelte` for usage, and if you want to see all the available font classes, take a look at this file of the project
https://github.com/James-revans/Photography-site/blob/master/src/assets/scss/base/_fonts.scss